### DISCUSSION TITLE
when copilot is enabled VSCode extension host stops.

### DISCUSSION THREAD

**Discussion by sanket-bhalerao** [community/community#6695] [Copilot Conversations]
whenever i enable copilot the VSCode extension host stops and nothing works.

**Comment by davecheney** [comment]
I'm sorry to hear that. Can you please give some details about your setup; which os, which version of vscode, which version of the copilot extension, etc. Can you also include screenshots or text of any errors or dialog boxes you receive.

**Reply by sanket-bhalerao** [reply]
I am using 
**OS**: WIndows 10
**copilot** v1.5.3378
**VSCode insiders Version**: 1.62.0-insider (user setup)
Commit: bcb9df5afa52c5b713f62e86dcac2c5050504b5f
Date: 2021-10-22T05:15:25.330Z
Electron: 13.5.1
Chrome: 91.0.4472.164
Node.js: 14.16.0
V8: 9.1.269.39-electron.0
OS: Windows_NT x64 10.0.19043

screenshot
![image](https://user-images.githubusercontent.com/9264813/138634697-76b92e98-6bba-4da4-95b7-d53c641687c0.png)

I get the issue if I enable Copilot. as soon as I disable Copilot and reload there is no issue and all the other extensions run fine.

**Reply by davecheney** [reply]
I see you're running the -insider build, could you try the non insider build and see if that helps/doesn't help.

**Reply by sanket-bhalerao** [reply]
sure let me try. 
is there any restriction with regards to using/not using Copilot with VSCode Insiders? (I use Insiders build as my primary editor)

**Reply by davecheney** [reply]
Not that I am aware of, but just like running any beta builds, you might occasionally encounter a speed bump.

**Reply by sanket-bhalerao** [reply]
the stable version seems to work fine

**Reply by davecheney** [reply]
Excellent. Thanks for confirming.

**Reply by sanket-bhalerao** [reply]
Insiders' build still has the issue.
not sure if I am the only one who is facing this issue.

**Reply by 9267aakashsharma** [reply]
I am facing the same issue as well with insiders build

**Comment by FollowTheProcess** [comment]
This seems to be linked to https://github.com/community/community/discussions/40370, I've got some more detailed debug info on that thread
